Doc/library/configparser.rst

```diff
@@ -1034,7 +1034,13 @@ ConfigParser Objects
       instance will contain an empty dataset.  An application which requires
       initial values to be loaded from a file should load the required file or
       files using :meth:`read_file` before calling :meth:`read` for any
-      optional files::
+      optional files
+
+      Raise a :exc:`ParsingError` instead of an :exc:`AttributeError`
+      when the configuration contains a key without a corresponding value.
+      This change is intended to handle scenarios where a key lacks a value
+      in a configuration. Please note that it's not recommended to have extra
+      spaces or blank lines within sections of the configuration.
```

Comment:
I don't think we need this same text repeated 4 times in the documentation. I don't have time right now to figure out where it actually belongs.

And I don't think this rises to the level of needing a ".versionchanged" entry.

Reply:
Sure, I will revert this